lib/sdr-client.rb

```diff
@@ -1,3 +0,0 @@
-# frozen_string_literal: true
-
-require 'sdr_client'
```

Comment:
None of our projects `require 'sdr-client'` anymore, so this is vestigial code and it's not zeitwerk-compliant, so throw it out.